tests/test_legendastv.py

```diff
@@ -333,3 +334,14 @@ def test_under_maintenance(movies):
             pass
         else:
             pytest.fail()
+
+
+@pytest.mark.integration
+def test_unrar_not_available(monkeypatch):
+    monkeypatch.setattr(rarfile, 'UNRAR_TOOL', 'test')
```

Comment:
If this is supposed to be a path to the unrar binary maybe you can replace `test` by something more meaningful?

Reply:
@Diaoul goal was to set an invalid path. I will change it